code/modules/surgery/advanced/dna_recovery.dm

```diff
@@ -15,7 +15,7 @@
 	if(!..())
 		return FALSE
 
-	if(!HAS_TRAIT_FROM(target, TRAIT_BADDNA, CHANGELING_DRAIN))
+	if(!HAS_TRAIT_FROM(target, TRAIT_BADDNA, CHANGELING_DRAIN) || !HAS_TRAIT(target, TRAIT_HUSK))
```

Comment:
Isnt this supposed to be an AND instead of an OR

Reply:
Fuc